src/translations/en.json

```diff
@@ -1113,6 +1113,12 @@
           },
           "theme": {
             "theme": "Theme (Optional)"
+          },
+          "card": {
```

Comment:
thoughts on this translation strategy? could this be simpler? would add this for each config option

Reply:
You could create a separate key for optional and required, so we don't have to translate every key twice.

```json
{
"entity": "Entity",
"required": "Required",
"optional": "Optional"
}
```
```js
`${this.hass.localize("ui.panel.lovelace.editor.card.entity")} (${this.hass.localize("ui.panel.lovelace.editor.required")})`
```